From: Andreas Stieger <andreas.stieger@gmx.de>
Date: 2014-10-21 22:42:56 +0100
Subject: Correct comment style
Upstream: Committed
References: https://code.google.com/p/serf/source/detail?r=2443

Fixes error:
test/test_buckets.c: In function 'deflate_compress':
test/test_buckets.c:1237:1: error: expected expression before '/' token
 //    buf_size = orig_len + (orig_len / 1000) + 12;
 ^

------------------------------------------------------------------------
r2443 | andreas.stieger@gmx.de | 2014-10-21 22:42:56 +0100 (Tue, 21 Oct 2014) | 7 lines
Changed paths:
   M /trunk/test/test_buckets.c

Follow-up to r2419: Correct comment style

* test/test_buckets.c
  (deflate_compress): Correct comment style not supported by strict
  compilers, fails on /branches/1.3.x


------------------------------------------------------------------------
Index: trunk/test/test_buckets.c
===================================================================
--- trunk/test/test_buckets.c	(revision 2442)
+++ trunk/test/test_buckets.c	(revision 2443)
@@ -1323,9 +1323,9 @@ static apr_status_t deflate_compress(const char **
 
     /* The largest buffer we should need is 0.1% larger than the
        uncompressed data, + 12 bytes. This info comes from zlib.h.
+       buf_size = orig_len + (orig_len / 1000) + 12;
        Note: This isn't sufficient when using Z_NO_FLUSH and extremely compressed
        data. Use a buffer bigger than what we need. */
-//    buf_size = orig_len + (orig_len / 1000) + 12;
     buf_size = 100000;
 
     write_buf = apr_palloc(pool, buf_size);
